llvm/lib/SYCLLowerIR/LowerESIMD.cpp

```diff
@@ -982,6 +1056,11 @@ static void createESIMDIntrinsicArgs(const ESIMDIntrinDesc &Desc,
       GenXArgs.push_back(UndefValue::get(Ty));
       break;
     }
+    case ESIMDIntrinDesc::GenXArgRuleKind::CONST_INT8: {
+      auto Ty = IntegerType::getInt8Ty(CI.getContext());
+      GenXArgs.push_back(llvm::ConstantInt::get(Ty, Rule.I.ArgConst));
+      break;
+    }
```

Comment:
Is it possible to add a test? Otherwise LGTM, as long as Denis's comments are addressed

Reply:
Added test